doc/MANUAL.md

```diff
@@ -660,6 +624,54 @@ indicates that CVEs may be present in the code.  A good result here is 0.
 
 The logging modes provide additional fine-grained control for debug information.
 
+## Merge Report Arguments
+
```

Comment:
```suggestion

Users may wish to create and combine multiple cve-bin-tool reports to track how vulnerability changes over time, how long it takes to fix issues, or other changes between different reports.  We have a number of options related to merging report data.

```

Taking a stab at an intro section with some common usages, mostly for users wondering if they should try out these arguments.